mpifileutils.rpmlintrc

```diff
@@ -4,3 +4,5 @@ from Config import *
 # https://github.com/rpm-software-management/rpmlint/issues/73
 addFilter(r'mpifileutils-.+-devel\..+: W: only-non-binary-in-usr-lib')
 
+# https://github.com/hpc/mpifileutils/issues/567
+addFilter(r'^mpifileutils-.+\.x86_64: E: missing-call-to-setgroups-before-setuid \/usr\/lib64\/.+\/bin\/dcp')
```

Comment:
I think this is okay since we don't have 100% control over the source code

Reply:
Ideall, we should be filing a ticket upstream to get this fixed there.

Reply:
If need be, I could push a fix myself and have it reviewed and merged by the repo maintainer

Reply:
Well, on the one hand, it would be nice to get this fixed as it just makes our environment more secure by doing so, even though it's just a test environment.  But given that we do not distribute this package I don't think it falls under an SDL requirement.  Perhaps an MSS, but I don't think MSS gets as detailed as that.

So, would be nice to have fixed, but is not a blocker of any sort for us.

Reply:
I do have another mpifileutils update coming, so I can fix this upstream before I update this package again